src/warp/cli/commands.py

```diff
@@ -4,7 +4,6 @@
 from typing import Any, Dict, Optional, Sequence, Union
 
 import aiohttp
```

Comment:
You can now remove the import of `aiohttp` from here, and from the dependencies!